Documentation/network/l2-announcements.rst

```diff
@@ -596,6 +596,30 @@ To enable L2 Pod Announcements, set the following:
             enable-l2-pod-announcements: true
             l2-pod-announcements-interface: eth0
 
+The ``l2podAnnouncements.interface``/``l2-pod-announcements-interface`` options allows you to specify 
+one interface.  If you would like to send on multiple interfaces, you should use the
```

Comment:
Nit (non-blocking):

“... allows you to specify one interface.” -> To do what? Maybe “... to use to send announcements”?